packages/model-fragments/lib/fragments/model.js

```diff
@@ -241,5 +135,29 @@ function getActualFragmentType(declaredType, options, data) {
   return actualType || declaredType;
 }
 
+// Returns the internal model for the given record/fragment
+export function internalModelFor(record) {
+  var internalModel = record._internalModel;
+
+  // Ensure the internal model has a fragments hash, since we can't override the
+  // constructor function anymore
+  if (!internalModel._fragments) {
+    internalModel._fragments = create(null);
+  }
+
+  return internalModel;
+}
```

Comment:
On a related note, methods like `Ember.create` are deprecated, and ED is using its own polyfills to avoid those warnings for v1.13, but they aren't exposed globally. Just copy-pasting them into this project seems wasteful, but I don't really see another way to prevent deprecation warnings – do you have any suggestions?